src/test/java/unit/AuthLoginLogout.java

```diff
@@ -64,14 +58,35 @@ void loginTestUser() throws IOException {
         equalToIgnoringCase(currentTerraUser.get().terraUserEmail));
   }
 
+  @Test
+  @DisplayName("test user logout updates global context")
+  void logoutTestUser() throws IOException {
+    // select a test user and login
+    TestUsers testUser = TestUsers.chooseTestUser();
+    testUser.login();
+
+    // `terra auth revoke`
+    TestCommand.Result cmd = TestCommand.runCommand("auth", "revoke");
+    assertEquals(0, cmd.exitCode);
+
+    // check that the credential store on disk is empty
+    DataStore<StoredCredential> dataStore = TestUsers.getCredentialStore();
+    assertEquals(0, dataStore.keySet().size(), "credential store is empty");
```

Comment:
You can also do `assertThat(collection, is(empty()))`. I don't love Hamcrest though.

Reply:
Yeah I'm not a huge fan of all those parentheses. I think it makes it slightly harder to read. I'll save Hamcrest for when it really makes things easier to read (e.g. `nullOrBlankString` is a nice one).